server/src/main/java/com/linecorp/centraldogma/server/metadata/PerRolePermissions.java

```diff
@@ -73,14 +73,14 @@ public static PerRolePermissions ofDefault() {
      * Creates a {@link PerRolePermissions} which allows accessing a repository from everyone.
      */
     public static PerRolePermissions ofPublic() {
-        return new PerRolePermissions(READ_WRITE, READ_WRITE, READ_WRITE);
+        return new PerRolePermissions(READ_WRITE, READ_WRITE, READ_WRITE, READ_ONLY);
```

Comment:
Let's remove the read permission to anonymouse users by default. 😉 
```suggestion
        return new PerRolePermissions(READ_WRITE, READ_WRITE, READ_WRITE, NO_PERMISSION);
```